packages/core/src/i18n/translations/tl.js

```diff
@@ -1,3 +1,15 @@
 export default {
+  'error': 'Error',
+  'required': '(*Kailangan)',
+  'max-chars': '(Pinakamarami ang {{length}} character)',
```

Comment:
That's [what we received](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1282) so I'm assuming it's correct:

```
{
"error": "Error",
"required:": "Kailangan",
"max-x-characters": "Pinakamarami ang {{number}} character",
"date-of-birth": "Petsa ng Kapanganakan",
"month": "Buwan",
"day": "Araw",
"year": "Taon",
"please-enter-a-year-between-x-and-y": "Mangyaring ilagay ang taon sa pagitan ng {{year1}} at {{year2}}",
"collapse-all-accordions": "I-collapse ang lahat ng accordion",
"expand-all-accordions": "I-expand ang lahat ng accordion",
"collapse-all": "I-collapse ang lahat",
"expand-all": "I-expand lahat"
}
```

Reply:
Hmm. Google Translate reports that "character" translates to a number of English words but the definitions of all those are related to other meanings of the English word character. So it may be that there is no direct translation. We could send that back to them though for future consideration in future (we're awaiting other translations).